packages/core/src/extensions/BubbleMenu/component/BubbleMenu.tsx

```diff
@@ -22,42 +22,32 @@ import { findBlock } from "../../Blocks/helpers/findBlock";
 import { formatKeyboardShortcut } from "../../../utils";
 import LinkToolbarButton from "./LinkToolbarButton";
 import { IconType } from "react-icons";
+import { Node } from "prosemirror-model";
 
-type ListType = "li" | "oli";
-
-function getBlockName(
-  currentBlockHeading: number | undefined,
-  currentBlockListType: ListType | undefined
-) {
-  const headings = ["Heading 1", "Heading 2", "Heading 3"];
-  const lists = {
-    li: "Bullet List",
-    oli: "Numbered List",
-  };
-  // A heading that's also a list, should show as Heading
-  if (currentBlockHeading) {
-    return headings[currentBlockHeading - 1];
-  } else if (currentBlockListType) {
-    return lists[currentBlockListType];
-  } else {
+function getBlockName(blockContentNode: Node) {
```

Comment:
we should find a way to define this as part of the block / node. But can also do that when we work on the Block API

Reply:
This function can be replaced by `getBlockInfoFromPos`, but I think we also want to find a better way of integrating that function into the block code.

Reply:
I think we should wait until we start on the API, since `getBlockInfoFromPos` is another useful function defined outside `Block.tsx` that we want users to have access to, and I'm not sure yet where to put it either.